NavCog3.xcodeproj/project.pbxproj

```diff
@@ -467,6 +467,22 @@
+		E201FC5E2875E17700A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC602875E81100A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC622875E82600A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC77287863C500A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC79287863D900A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E2082D1E28B4B9F2008492D1 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/Fingerprint.strings; sourceTree = "<group>"; };
+		E2301B7727FF74C800A62C75 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/HLPGeoJSON.strings; sourceTree = "<group>"; };
+		E2301B7827FF74D100A62C75 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/BlindView.strings; sourceTree = "<group>"; };
+		E2301B7927FF74E700A62C75 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.stringsdict; name = ar; path = ar.lproj/BlindView.stringsdict; sourceTree = "<group>"; };
+		E2301B7A27FF74F500A62C75 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/Localizable.strings; sourceTree = "<group>"; };
+		E2301B7B27FF751E00A62C75 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/InfoPlist.strings; sourceTree = "<group>"; };
+		E262DF9D28807750007BC3D5 /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E2997D0E29327D6800ABB819 /* dev.hashfile */ = {isa = PBXFileReference; fileEncoding = 4; indentWidth = 4; lastKnownFileType = text; path = dev.hashfile; sourceTree = "<group>"; tabWidth = 4; usesTabs = 0; };
+		E2D0AAFE27FB893C0018AA63 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/Main.strings; sourceTree = "<group>"; };
+		E2D0AAFF27FB893C0018AA63 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/LaunchScreen.strings; sourceTree = "<group>"; };
+		E2D0AB0027FB893C0018AA63 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = SettingUtil/ar.lproj/HLPSettingView.strings; sourceTree = "<group>"; };
```

Comment:
`SettingUtil/ar.lproj/HLPSettingView.strings` this file is not included in this PR

Reply:
Done